MessageDialog.sln

```diff
@@ -77,11 +84,36 @@ Global
 		{58D94E26-9A4A-45AD-9F5E-2FCB9B2F0EAE}.Release|NuGet.Build.0 = Release|Any CPU
 		{58D94E26-9A4A-45AD-9F5E-2FCB9B2F0EAE}.Release|UAP.ActiveCfg = Release|Any CPU
 		{58D94E26-9A4A-45AD-9F5E-2FCB9B2F0EAE}.Release|UAP.Build.0 = Release|Any CPU
+		{D20A05F5-1775-4EFB-9EBA-882B162F2BAA}.Debug|Android.ActiveCfg = Debug|Any CPU
```

Comment:
@carlh98  @GabrielChouinardLetourneau  needed?

Reply:
Don't know if it is really necessary, I will resolve this conversation and let CH give his input